app/admin/book.rb

```diff
@@ -0,0 +1,9 @@
+ActiveAdmin.register Book do
+  permit_params :title, :published_date, :author, :price, :category
+
+  config.clear_action_items!
+
+  action_item :create, :only => :index do
```

Comment:
don't forget to use ruby 2.0 style hash syntax (ie don't use `=>`):

`action_item :create, only: :index do`